plugin.xml

```diff
@@ -44,7 +46,7 @@
     <source-file src="src/android/com/outsystems/plugins/barcodescanner/res/layout-land/activity_custom_scanner.xml" target-dir="res/layout-land"/>
     <source-file src="src/android/com/outsystems/plugins/barcodescanner/res/layout-land/custom_barcode_scanner.xml" target-dir="res/layout-land"/>
     
-    <framework src="com.github.OutSystems:zxing-android-embedded:teste" />
+    <framework src="com.github.OutSystems:zxing-android-embedded:v5.0.0" />
```

Comment:
Was this the dependency that we talked about in the scrum today? Where we said it may be better to include our versioning system (v5.0.0-OS) ?

Reply:
I will change it to use the .jar or change to OS release

Reply:
but for now I will keep this so do not create a final release after the tests